libs/algolia/useFacets.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
```

Comment:
I think we usually avoid `import *` to reduce the number of potential naming conflicts, and make it easier to see where things are being used, but I'm not too familiar with TS conventions or best practices when importing third party libs like this so feel free to ignore.

Reply:
`import * as React from 'react'` is recommended by React core team ([see this](https://twitter.com/dan_abramov/status/1308739731551858689?s=20)).

That being said, if you are referring to named exports vs default exports then the reason I do this is to make it easier to see that I'm using a React primitive vs a custom primitive, but that's just my opinion